playground/src/main.ts

```diff
@@ -178,6 +178,9 @@ const shareCode = async (writeToClipboard?: boolean) => {
   const url = `${window.location}?${encoded}`;
   if (writeToClipboard) {
     await navigator.clipboard.writeText(url);
+    shareButtonEl.classList.remove("copied");
+    void shareButtonEl.offsetWidth; // trigger reflow
+    shareButtonEl.classList.add("copied");
```

Comment:
I don't love the "trigger reflow" approach, but seems to be the way

Source: https://stackoverflow.com/a/45036752/614112